pycbc/events/coinc.py

```diff
@@ -594,11 +592,22 @@ def __init__(self, num_templates, analysis_block, background_statistic,
         from . import stat
         self.num_templates = num_templates
         self.analysis_block = analysis_block
+
+        # Only pass a valid stat file for this ifo pair
+        for fname in stat_files:
+            f = h5py.File(fname, 'r')
```

Comment:
doesn't h5py need to be imported before this?

Reply:
Yes, done.